fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/AbstractCumulativeLoanScheduleGenerator.java

```diff
@@ -68,6 +68,11 @@ public LoanScheduleModel generate(final MathContext mc, final LoanApplicationTer
         return generate(mc, loanApplicationTerms, loanCharges, holidayDetailDTO, null);
     }
 
+    @Override
+    public LoanScheduleModel generate(final MathContext mc, final LoanRepaymentScheduleModelData modelData) {
+        return generate(mc, modelData);
```

Comment:
Is this call itself recursively?

Reply:
Yes, I forgot to delete this... This code has already been deleted. Thank you